pkg/yurtctl/constants/constants.go

```diff
@@ -144,7 +144,9 @@ spec:
       serviceAccountName: yurt-controller-manager
       hostNetwork: true
       tolerations:
-      - operator: "Exists"
+      - key: "node-role.kubernetes.io/master"
+        effect: ""
+        operator: "Exists"
```

Comment:
Please comment the reason of adding key: "node-role.kubernetes.io/master"?

Reply:
same as previous one